beta/experience-platform-extension/tutorials/README.md

```diff
@@ -25,3 +25,21 @@ Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https:/
 * [ ] XDM implementation validation
 * [ ] Clarity of documentation during the beta review call
 
+
+### Assignment 2
+
+In this assignment you will learn how to:
+
+1. Install Swfit Core library using Cocoapods
+2. Initialize SDK with new API
+3. Start/Stop lifecycle
+4. Make an Identity call
+
+Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
+
+#### Prepare to provide feedback on:
+
+* [ ] Overall setup and implementation workflow of AEPCore SDK
+* [ ] In Pod file, instead of just include ACPCore, now you need to cinlude AEPServcies, AEPCore, AEPLifecyle, AEPIdentity, AEPSignal, does that bother you?
+* [ ] SPM vs Cocoapod, which dependency mangament tool are you using today?
```

Comment:
probably better to use a hyphen here than a comma:

`SPM vs. Cocoapods - which dependency...`

probably also good to ask if they are considering adopting or changing dependency management tools in the near future